### ISSUE TITLE
Feature - run tests on change

### ISSUE THREAD

**Issue by jenweber** [thefrontside/bigtest#822]
This issue requires team discussion before it can be worked on.

When I was writing my first BigTest, I found myself wishing that I had a way to automatically re-run tests after a change. I did not have to restart the BigTest server, but I did have to relaunch the test each time. A `watch` mode is a feature that users may be expecting from their past experience with Jest or Ember.

**Comment by jnicklas** [comment]
Sounds like a great idea, and should not be too difficult to implement.

**Comment by wKich** [comment]
I agree the `watch` mode is a must-have feature. The initialization time of BigTest, especially with building an app, might be too much
